__tests__/news.test.js

```diff
@@ -250,6 +250,52 @@ describe('GET /api/articles/:article_id/comments', () => {
         expect(responseArray).toEqual("Not Found");
         });
     });
+    test('should return appropriate error if invalid input data type', () => {
+        return request(app)
+        .get(`/api/articles/TEXT/comments`)
+        .expect(400)
+        .then((response) => {
+            const responseArray = response.res.statusMessage
+            expect(responseArray).toEqual("Bad Request");
+          })
+    });
 
 });
 
+describe('POST /api/articles/:article_id/comments', () => {
+    test('should return posted comment', () => {
+        return request(app)
+        .post('/api/articles/3/comments').send({username: 'butter_bridge', body: "a good old body of text"})
+        .expect(201)
+        .then((response) => {
+            const responseComment = response.body.comment
+            expect(responseComment).toMatchObject({
+                comment_id: expect.any(Number),
+                body: "a good old body of text",
+                article_id: 3,
+                author: "butter_bridge",
+                votes: 0,
+                created_at: expect.any(String)
+            })
+        })
+        
+    });
+    test('should return appropriate error if unacceptable request body', () => {
+        return request(app)
+        .post('/api/articles/3/comments').send({body: 'I am an acceptable body of text shame about the other guy', unacceptableKey: 'I am so unacceptable'})
```

Comment:
I think this should fail more specifically because there is no username and not because there is a key we don't recognise.